Content.Shared/MedicalScanner/SharedMedicalScannerComponent.cs

```diff
@@ -17,25 +17,25 @@ public abstract class SharedMedicalScannerComponent : Component, IDragDropOn
         public class MedicalScannerBoundUserInterfaceState : BoundUserInterfaceState
         {
             public readonly EntityUid? Entity;
-            public readonly Dictionary<DamageClass, int> DamageClasses;
-            public readonly Dictionary<DamageType, int> DamageTypes;
+            public readonly Dictionary<DamageGroupPrototype, int> DamageGroup;
+            public readonly Dictionary<DamageTypePrototype, int> DamageTypes;
```

Comment:
Use string ids here instead of sending the prototype, I don't think the technology is quite there yet.